apps/journeys-admin/src/components/Editor/utils/useBlockCreateCommand/useBlockCreateCommand.tsx

```diff
@@ -0,0 +1,78 @@
+import { useCommand } from '@core/journeys/ui/CommandProvider'
+import { ActiveSlide, useEditor } from '@core/journeys/ui/EditorProvider'
+import { TreeBlock } from '@core/journeys/ui/block'
+import { BlockFields } from '../../../../../__generated__/BlockFields'
+import { BlockRestore_blockRestore as RestoredBlock } from '../../../../../__generated__/BlockRestore'
+import { useBlockDeleteMutation } from '../../../../libs/useBlockDeleteMutation'
+import { useBlockRestoreMutation } from '../../../../libs/useBlockRestoreMutation'
+
+interface AddBlockParameters {
+  execute: () => Promise<void>
```

Comment:
don't care if promise or not.

```suggestion
  execute: () => void
```